pulsar-broker/src/main/java/org/apache/pulsar/broker/intercept/BrokerInterceptor.java

```diff
@@ -231,44 +231,6 @@ default void onFilter(ServletRequest request, ServletResponse response, FilterCh
      */
     void initialize(PulsarService pulsarService) throws Exception;
 
-    BrokerInterceptor DISABLED = new BrokerInterceptorDisabled();
```

Comment:
Is the motivation for removing this disabled interceptor to simply make it clear when we do not have an interceptor?

Reply:
Exactly. I didn't want to add a special way to detect that.